### ISSUE TITLE
Triplet Loss + Arcface

### ISSUE THREAD

**Issue by onlinehuazai** [ronghuaiyang/arcface-pytorch#102]
Why (Triplet loss + Arcface) is inferior than (Triplet loss + Softmax), I have tried many parameters, but the retrieval performance is not well.  Triplet loss + Softmax is always better.

**Comment by onlinehuazai** [comment]
> Why (Triplet loss + Arcface) is inferior than (Triplet loss + Softmax), I have tried many parameters, but the retrieval performance is not well. Triplet loss + Softmax is always better.

On my dataset, I think Arcface is better than Softmax, so (Triplet loss + Arcface) is better, but the facts present I am wrong.

**Comment by chuong98** [comment]
Have the same issue and I can't explain this problem neither.
